android/src/main/java/com/sensors/Accelerometer.java

```diff
@@ -44,6 +39,10 @@ public void setUpdateInterval(int newInterval) {
 
   @ReactMethod
   public void startUpdates() {
+    if (this.sensor == null) {
+      // No sensor found, throw error
+      throw new RuntimeException("No Accelerometer found");
```

Comment:
Can we use `RCTMakeError` instead of the RuntimeException? I think following the discussion in #8 this would be the best solution to not crash the app completely, but we can catch it in the JS